tests/test_build_html.py

```diff
@@ -421,6 +421,12 @@ def test_html5_output(app, cached_etree_parse, fname, expect):
     check_xpath(cached_etree_parse(app.outdir / fname), fname, *expect)
 
 
+@pytest.mark.skipif(docutils.__version_info__ < (0, 13),
+                    reason='docutils-0.13 or above is required')
```

Comment:
Why are you marking this as docutils-0.13 or above only?

Reply:
I copied the example below. I don't have any understanding of why it might be needed. Should I remove it?

Reply:
Yes, please. The above and below test cases surely depends on docutils-0.13+. But this is not.

Reply:
Removed unnecessary skipif